src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -47,24 +47,29 @@ object Boot extends App {
     }
 
     val submissionSupervisor = system.actorOf(SubmissionSupervisor.props(
-      new GraphWorkspaceDAO(),
-      new GraphSubmissionDAO(new GraphWorkflowDAO()),
+      GraphContainerDAO(
+        new GraphWorkflowDAO(),
+        new GraphWorkspaceDAO(),
+        new GraphEntityDAO(),
+        new GraphMethodConfigurationDAO(),
+        new GraphSubmissionDAO(new GraphWorkflowDAO())
+      ),
       new HttpExecutionServiceDAO(conf.getConfig("executionservice").getString("server")),
-      new GraphWorkflowDAO(),
-      new GraphEntityDAO(),
-      new GraphMethodConfigurationDAO(),
       dataSource
     ).withDispatcher("submission-monitor-dispatcher"), "rawls-submission-supervisor")
 
     val service = system.actorOf(RawlsApiServiceActor.props(
                     WorkspaceService.constructor(dataSource,
-                                                  new GraphWorkspaceDAO(),
-                                                  new GraphEntityDAO(),
-                                                  new GraphMethodConfigurationDAO(),
+                                                  GraphContainerDAO(
+                                                    null,
+                                                    new GraphWorkspaceDAO(),
+                                                    new GraphEntityDAO(),
+                                                    new GraphMethodConfigurationDAO(),
+                                                    new GraphSubmissionDAO(new GraphWorkflowDAO)
+                                                  ),
```

Comment:
only create 1 GraphContainerDAO and pass it to both submission supervisor and workspace service